packages/schematics/migrations/update-8-0-0/update-8-0-0.ts

```diff
@@ -32,12 +32,14 @@ function addDependencies() {
     const dependencies = readJsonInTree(host, 'package.json').dependencies;
     const builders = new Set<string>();
     const projects = readJsonInTree(host, 'angular.json').projects;
-    Object.values<any>(projects).forEach(project => {
-      Object.values<any>(project.architect).forEach(target => {
-        const [builderDependency] = target.builder.split(':');
-        builders.add(builderDependency);
+    Object.values<any>(projects)
+      .filter(project => typeof project === 'object')
```

Comment:
If I understand the issue properly, you need this in order to solve the issue.

```suggestion
      .filter(project => typeof project === 'object' && project.hasOwnProperty('architect'))
```

Reply:
Thank you for point-out!
I will fix it.

Reply:
I understand the problem and corrected it!